File: tjhsst comp sci 2/Java/Units/Unit1/Lab09.java
//Name: Vivian Feng           Date: March 18,2020

import edu.fcps.karel2.Display;
import edu.fcps.karel2.Robot;
import javax.swing.JOptionPane;

public class Lab09
{
   public static void main(String[] args)
   {
      String filename = JOptionPane.showInputDialog("Which world?");
      Display.openWorld("maps/"+filename+".map");
      Display.setSize(10,10);
      Display.setSpeed(10);
      
      Athlete a = new Athlete(1,1,Display.EAST,Display.INFINITY);
      
      int[] countlist = new int[7];
      for (int i=0;i<7;i++)
      {
         int x = 0;
         while (a.nextToABeeper())
         {
            a.pickBeeper();
            x++;
         }
         countlist[i] = x;
         a.move();
      }
      a.turnAround();
      for (int j=0;j<7;j++)
      {
         a.move();
      }
      a.turnAround();
      a.move();
      
      for (int count:countlist)
      {
         for (int g=0;g<count;g++)
         {
            a.putBeeper();
         }a.move();
      }
   }


}

File: tjhsst comp sci 2/pyKarel Student Code/Lab09.py
#Name: Vivian and Karthik
#Date: 9/13/2019


from pyKarel import *
from Athlete import *

wld_choice = raw_input("Which world? ")
wld = World(wld_choice,height=3,width=8)
a = Athlete(wld,1,1,east,0)

count1 = 0
count2 = 0
count3 = 0
count4 = 0
count5 = 0
count6 = 0
count7 = 0

countlist = [count1,count2,count3,count4,count5,count6,count7]
for i in range(7):
   x = 0
   while a.nextToABeeper():
      a.pickBeeper()
      x+=1
   countlist[i] = x
   a.move()
                           
   
a.turnAround()
for j in range(7):
   a.move()

a.turnAround()
a.move()
for count in countlist:
   for g in range(count):
      a.putBeeper()
   a.move()

wld.mainloop()
